lib/config/presets/internal/group.ts

```diff
@@ -444,6 +445,55 @@ const staticGroups = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  remark: {
+    description: 'Group remark packages together.',
+    matchSourceUrls: [
```

Comment:
I think you should use source url prefix with org scope, this rule will be combined with and to the match package prefixes

Reply:
Not sure if I'm understanding what you mean, but I added this commit:

47f9c8cf021d6c3144d122c47dc7d8e89c0bb4ab

Reply:
Maybe you mean this change instead:

- Remove matchPackagePrefixes 5f4864d